__tests__/integration.test.ts

```diff
@@ -8,14 +9,26 @@ jest.mock('@actions/core')
 const MOCK_NAMESPACE = 'GithubCI'
 const MOCK_REPO = 'ros-tooling/action-cloudwatch-metrics'
 const MOCK_RETURN = 'success'
+const ENVIRONMENT_VARIABLE_OVERRIDES = {
+  GITHUB_REPOSITORY: 'MY-REPOSITORY-NAME',
+  GITHUB_WORKFLOW: 'MY-WORKFLOW-ID',
+  GITHUB_ACTION: 'MY-ACTION-NAME',
+  GITHUB_ACTOR: 'MY-USERNAME[bot]',
```

Comment:
[bot]?

Reply:
Just an example of a bot user, like greenkeeper (Covers all cases down the line).